### PR TITLE
Greek Language

### PR DESCRIPTION
**Pull request by psavva**
Please find the last of the translations for Greek.
I hope it all works :)

### CODE REVIEW

**Comment by thinkyhead** [comment]
@psavva We need to try to keep messages below 20 characters. Anything after 20 characters is just wasted – that part of the string will never be displayed, and just wastes space.

Ideally try to keep strings under 18 characters (for menu items).

Strings of 16 characters or fewer are the best. They are guaranteed to be visible on all displays, as the minimum width for displays is 16 characters.

**Comment by psavva** [comment]
Ok.  Back to the drawing board.  I'll see how I can shorten it without losing its meaning...

**Comment by thinkyhead** [comment]
> I'll see how I can shorten it without losing its meaning

That is the real challenge! The Germans spent weeks on it.

File: Marlin/language_el.h [hunk 1]
```diff
@@ -170,27 +170,27 @@
...
-#define MSG_ERR_MAXTEMP                     "Λάθος: ΜΕΓΙΣΤΗ ΘΕΡΜΟΤΗΤΑ" //SHORTEN
-#define MSG_ERR_MINTEMP                     "Λάθος: ΕΛΑΧΙΣΤΗ ΘΕΡΜΟΤΗΤΑ" //SHORTEN
-#define MSG_ERR_MAXTEMP_BED                 "Λάθος: ΜΕΓΙΣΤΗ ΘΕΡΜΟΤΗΤΑ ΚΛΙΝΗΣ" //SHORTEN
-#define MSG_ERR_MINTEMP_BED                 "Λάθος: ΕΛΑΧΙΣΤΗ ΘΕΡΜΟΤΗΤΑ ΚΛΙΝΗΣ" //SHORTEN
+#define MSG_ERR_REDUNDANT_TEMP              "ΠΛΕΟΝΑΖΟΥΣΑ ΘΕΡΜΟΤΗΤΑ"
+#define MSG_THERMAL_RUNAWAY                 "ΔΙΑΦΥΓΗ ΘΕΡΜΟΚΡΑΣΙΑΣ"
+#define MSG_ERR_MAXTEMP                     "ΠΕΡΙΤΗ ΘΕΡΜΟΚΡΑΣΙΑ"
+#define MSG_ERR_MINTEMP                     "ΜΗ ΕΠΑΡΚΗΣ ΘΕΡΜΟΚΡΑΣΙΑΣ" //SHORTEN
+#define MSG_ERR_MAXTEMP_BED                 "ΜΕΓΙΣΤΗ ΘΕΡΜΟΚΡΑΣΙΑΣ ΕΠ. ΕΚΤΥΠΩΣΗΣ" //SHORTEN
+#define MSG_ERR_MINTEMP_BED                 "ΕΛΑΧΙΣΤΗ ΘΕΡΜΟΚΡΑΣΙΑΣ ΕΠ. ΕΚΤΥΠΩΣΗΣ" //SHORTEN
```
Comment (thinkyhead) [inline Marlin/language_el.h:187]: Everything after character 20 is wasted space. You might as well concatenate.
I find it helps to use a "ruler" when working on strings:

```
12345678901234567890
Διερεύνηση Z εκτός Επ.Εκτύπωσης
ΜΕΓΙΣΤΗ ΘΕΡΜΟΚΡΑΣΙΑΣ ΕΠ. ΕΚΤΥΠΩΣΗΣ
ΕΛΑΧΙΣΤΗ ΘΕΡΜΟΚΡΑΣΙΑΣ ΕΠ. ΕΚΤΥΠΩΣΗΣ

12345678901234567890
Διερεύνηση Z εκτός Ε
ΜΕΓΙΣΤΗ ΘΕΡΜΟΚΡΑΣΙΑΣ
ΕΛΑΧΙΣΤΗ ΘΕΡΜΟΚΡΑΣΙΑ
```

**Comment by psavva** [comment]
@thinkyhead, I'm in the process of shortening everything over 20 characters.

It's still a WIP.

I'm committing to ensure I don't lose my changes.

Some cases, I'm getting suggestions on better wordings, but at the same time didn't figure out the best short version.

I have the list to shorten.  
Hopefully I will have it ready soon...

**Pull request merged**